README.md

```diff
@@ -1,115 +1,7 @@
-## OpenShift 4.4+ Designs
-Access design documentation for features slotted in the 4.4 release of OpenShift. Click links below for specific section details.
+## This repo is separated into the following sections
+- [Administration Console Designs](http://openshift.github.io/openshift-origin-design/admin-console/admin-console)
+- [Developer Console Designs](http://openshift.github.io/openshift-origin-design/dev-console/dev-console)
+- [Storage Designs (OpenShift Container Storage)](http://openshift.github.io/openshift-origin-design/storage/storage)
+- [Virtualization Designs (Kubevirt)](http://openshift.github.io/openshift-origin-design/virtualization/virtualization)
+- [OCM Designs (OpenShift Cluster Manager) )](http://openshift.github.io/openshift-origin-design/cluster-manager/cluster-manager)
```

Comment:
This should link to the same place the current README does: https://docs.google.com/document/d/13G1d9tvOhIz0DDt4NyuavhCUSTmPR3z1uUEQIpaVvAA/edit#

If at some point we decide to add OCM designs to GH, then I can create a directory.

Reply:
ok will do, ya I created a readme just in case in the future we wanted to populate it but we can just link straight to the doc if you think that's best.

Reply:
I think just the link is good. Otherwise it's just an added layer of misdirection.